### DISCUSSION TITLE
Example Guide

### DISCUSSION THREAD

**Discussion by ehan08** [OpenRadioss/OpenRadioss#483] [Ideas]
The example guide at https://2021.help.altair.com/2021/hwsolvers/rad/topics/solvers/rad/example_guide_overview_r.htm provides a lot of good examples. They should help the beginners to start and learn OpenRadioss. Is there a link to download the example input files? Thanks.

**Comment by elequiniou** [comment]
This is a very important point raised. Several of these examples are already available for the community, please have a look at https://www.openradioss.org/models/ and the confluence page which provides input files for each of them: https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/8749071/Example+Models
This is WIP and we will add more. Thanks!

**Comment by ehan08** [comment]
yeah, these examples are very helpful. The tensile test model can be modified to test variable material models and validate the settings. Small models are good for studying the features and learning the code. We may have a place for people to update and share such models.

Thanks.

**Reply by elequiniou** [reply]
Right, this is exactly what we have in mind to add soon, not only input but a place to share and update models for the community!
